pkg/util/wait/wait.go

```diff
@@ -133,12 +132,16 @@ var ErrWaitTimeout = errors.New("timed out waiting for the condition")
 // if the loop should be aborted.
 type ConditionFunc func() (done bool, err error)
 
+// sleepFn is abstracted for testing
+var sleepFn = time.Sleep
```

Comment:
This is no different than any of our other package scoped test code for stdlib functions?